### PR TITLE
Storing votes

### PR DESCRIPTION
**Pull request by Golemwardox**
For User Story #17

### CODE REVIEW

File: front-end/src/pages/ChooseCuisine.jsx [hunk 1]
```diff
@@ -3,26 +3,39 @@ import './ChooseCuisine.css';
...
     }
     cookies.set("cuisine", choice);
+    const response = await post(
+      '/random',
+      {
+        choice: choice,
+        groupId: cookies.get("groupID")
+      }
+    );
+    if(response.valid == true){
```
Comment (Jen-Lopez) [inline front-end/src/pages/ChooseCuisine.jsx:35]: `response.valid` would've sufficed here but it's all good!

File: back-end/routes/random.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+const router = express.Router();
+const Group = require("../models/group");
+
+router.post("/random", function (req, res) {
```
Comment (Jen-Lopez) [inline back-end/routes/random.js:5]: just to avoid confusion, can we rename this route to `/cuisine` since it applies to both choose/random pages?

**Review by Jen-Lopez — APPROVED** [review]
minor comments but 👍🏼

**Review by psc358 — APPROVED** [review]
LGTM!

**Review by Jen-Lopez — APPROVED** [review]

**Pull request merged**